tests/Doctrine/Tests/ORM/Functional/ExtraLazyCollectionTest.php

```diff
@@ -191,6 +191,62 @@ public function testCountOneToManyJoinedInheritance(): void
         self::assertCount(2, $otherClass->childClasses);
     }
 
+    public function testFirstWhenInitialized(): void
+    {
+        $user = $this->_em->find(CmsUser::class, $this->userId);
+        $this->getQueryLog()->reset()->enable();
+        $user->groups->toArray();
+
+        self::assertTrue($user->groups->isInitialized());
+        self::assertInstanceOf(CmsGroup::class, $user->groups->first());
+        $this->assertQueryCount(1, 'Should only execute one query to initialize collection, no extra query for first().');
```

Comment:
When running the test suite with second level cache enabled, this test fails (actual: 0). I'm not sure how to fix as I have no experience with the second level cache.

Reply:
I've added `@group non-cacheable` to this test as I see it on other similar tests in this file.